### PR TITLE
Jonmv/vespa cli test runner

### PR DESCRIPTION
**Pull request by jonmv**
@hakonhall please review and merge.

### CODE REVIEW

**Review by hakonhall — APPROVED** [review]

File: vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/VespaCliTestRunner.java [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
     @Inject
     public VespaCliTestRunner(VespaCliTestRunnerConfig config) {
-        this(config.artifactsPath());
+        this(config.artifactsPath().resolve("artifacts"));
```
[the remark this answers is not in the document]
Comment (jonmv) [reply vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/VespaCliTestRunner.java:46]: We can't change that.

**Comment by jonmv** [comment]
PTAL.

File: vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/VespaCliTestRunner.java [hunk 2]
```diff
@@ -133,8 +137,7 @@ private void log(Level level, String message, Throwable thrown) {
         }
     }
 
-    static String toEndpointsConfig(byte[] testConfig) throws IOException {
-        TestConfig config = TestConfig.fromJson(testConfig);
+    static String toEndpointsConfig(TestConfig config) throws IOException {
         Cursor root = new Slime().setObject();
         Cursor endpointsArray = root.setArray("endpoints");
         config.deployments().get(config.zone()).forEach((cluster, url) -> {
```
Comment (hakonhall) [inline vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/VespaCliTestRunner.java:143]: and scope

**Review by hakonhall — APPROVED** [review]

File: vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/VespaCliTestRunner.java (hunk 2 above)
Comment (jonmv) [reply vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/VespaCliTestRunner.java:143]: ?

Comment (hakonhall) [reply vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/VespaCliTestRunner.java:143]: You should add scope "zone" to each endpoint

Comment (jonmv) [reply vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/VespaCliTestRunner.java:143]: Ah. So we probably end up with custom parser code for this anyway, so we don't need those. See following PR.

Comment (hakonhall) [reply vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/VespaCliTestRunner.java:143]: No, my bad.  If the intention is anyway to just end up with the map, then no scope is needed.  I was thinking we needed to reuse the code that filters on scope, but there is no need for that.

**Pull request merged**